wagtail_wordpress_import/test/tests/test_commands.py

```diff
@@ -1,19 +1,80 @@
-from django.test import TestCase
+from unittest.mock import patch
+from django.test import TestCase, override_settings
+from django.core.management import call_command, CommandError
 from wagtail_wordpress_import.management.commands.import_xml import Command as RunCmd
 from wagtail_wordpress_import.management.commands.reduce_xml import Command as ReduceCmd
+from wagtail_wordpress_import.xml_boilerplate import (
+    build_xml_stream,
+    generate_temporay_file,
```

Comment:
**Suggestion**: There's a typo in this function name.

Reply:
I fixed this.